src/bin/orc-export.rs

```diff
@@ -0,0 +1,42 @@
+use std::{fs::File, io, path::PathBuf};
+
+use anyhow::Result;
+use arrow::csv;
+use clap::Parser;
+use orc_rust::ArrowReaderBuilder;
+
+#[derive(Parser)]
+#[command(name = "orc-export")]
+#[command(version, about = "Export data from orc file to csv", long_about = None)]
+struct Cli {
+    /// Path to the orc file
+    file: PathBuf,
+    /// Output file
```

Comment:
Maybe need to enhance doc here to indicate it outputs to stdout if not provided?

Reply:
fixed